index.js

```diff
@@ -1,12 +1,18 @@
 var express = require('express');
-var cors=require('cors')
-var people=require('./people')
-
 var app = express();
+var connection = require('./people/methods/dbConnection')
```

Comment:
dbConnection is not people's property neither people's concerns to connect it should be the server's responsibility remove / refactor this